react-native/realm.json

```diff
@@ -1,3 +1,4 @@
 {
-  "appId": "<Your App ID>"
+  "app_id": "todo-sync-tmikv",
```

Comment:
Did you mean to include your app id?

Reply:
Yes, it makes it easier for us to test and will be overridden by the generator.